### PR TITLE
Update national-institute-of-technology-karnataka.csl

### PR DESCRIPTION
**Pull request by biosainik**
Something wrong in the script. Trying to find out what.

### CODE REVIEW

**Comment by biosainik** [comment]
@rmzelle and @adam3smith  please help me merging the commits to the main branch

**Comment by rmzelle** [comment]
> Something wrong in the script. Trying to find out what.

Is there anything specific you're struggling with? I see that you created and close #2369. This pull request here might be okay, except for the fact that your proposed version is written on a single line (and we want our CSL styles to be properly indented).

(in general, we prefer it if you fix your original pull request instead of creating new ones, but we can continue here)

**Comment by biosainik** [comment]
@rmzelle After I started using csl style created from GitHub, my citation format has gone wrong. Later, today I found out that the code in one line work well. 
Can you please explain me why this happens...

**Comment by rmzelle** [comment]
> Can you please explain me why this happens...

Well, you have to describe in more detail what you're doing and what step is giving you problems.

**Comment by rmzelle** [comment]
I just reformatted your style with our http://formatter.citationstyles.org/ tool.

File: national-institute-of-technology-karnataka.csl [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" default-locale="en-GB">
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never">
```
Comment (rmzelle) [inline national-institute-of-technology-karnataka.csl]: @biosainik, will this CSL style be used in multiple languages, or just one?

Comment (rmzelle) [inline national-institute-of-technology-karnataka.csl]: If you expect the style to only be used in British English, it's better to restore the `default-locale="en-GB"` on the `<style>` tag. That way users won't have to select the language by hand each time.

(you can update this pull request through the instructions given above: https://github.com/citation-style-language/styles/pull/2370#issuecomment-267961802)

**Comment by rmzelle** [comment]
@biosainik, if you now look at https://github.com/citation-style-language/styles/pull/2370/files, you can see that the only change you're currently submitting is a new date stamp.

What were you trying to change with this pull request?

**Comment by biosainik** [comment]
@rmzelle. Of course, there should not be any other change except the date stamp. Though indentation of code creates the problem.
I hope you have read the following in the previous thread itself.
@rmzelle . there is no problem with system locale.
While writing code in the indented form, the problem is with the single quote appearing instead of double quote for the title (Zheng, C. (2014). **‘Characterization of natural........ **) and at the end of the title comes the full stop followed by a closing double quote.

**Comment by biosainik** [comment]
@rmzelle ..  If you have time, can you please try out the .csl file with and without indentation. It will help us to understand the problem.

**Comment by rmzelle** [comment]
> can you please try out the .csl file with and without indentation.

You mean the one-liner you originally submitted and my re-indented version? They should behave identically.

**Comment by biosainik** [comment]
No, Sir. They behave differently..

**Comment by biosainik** [comment]
My above observation is wrong.   Including demote-non-dropping-particle="never"  solves the problem, hopefully.

**Comment by biosainik** [comment]
Also, your suggest 'You can achieve that by either setting the style to US English (default-locale="en-US")' works fine.

Shall I commit this change and upload?

**Comment by rmzelle** [comment]
> Including demote-non-dropping-particle="never" solves the problem, hopefully.

Can you add that back in, then?

You original one-liner had `demote-non-dropping-particle="never"`, as did my re-indented version. You just removed it again when you put back the "default-locale" attribute (see https://github.com/citation-style-language/styles/pull/2370/commits/b4aabfdc5b242c186cff21b4135c56bfb1728f28).

**Comment by rmzelle** [comment]
> Shall I commit this change and upload?

Yes, please.

**Comment by rmzelle** [comment]
It's rather late here. I assume you still want to put back `demote-non-dropping-particle="never"` as well?

When you're done with the style and it behaves correctly, please confirm that it does, and I'll merge this tomorrow.

**Comment by biosainik** [comment]
It works fine now, Sir. You may please go ahead with the locale change to US

**Comment by biosainik** [comment]
Thanks a lot for the late night commitment for me. 

Regards,
Pragadeesh.

**Comment by rmzelle** [comment]
No problem, and thanks!

**Pull request merged**